packages/react-i18n/src/i18n.ts

```diff
@@ -499,9 +500,11 @@ export class I18n {
     const timeZone = options?.timeZone;
     const time = this.formatDate(date, {
       ...options,
+      ...undefinedDateDisplayProps,
```

Comment:
It's almost always a better pattern to explicitly pick the properties that are relevant instead of removing properties you don't want. That way, the contract is clear. We should probably do a destructing assignment from the options object to pick out the properties that are allowed to passed down and recompose the options object with those.